apps/builder/app/builder/features/topbar/publish.tsx

```diff
@@ -554,24 +571,40 @@ const ExportContent = () => {
             Step 3
           </Text>
           <Text color="subtle">
-            Use the{" "}
+            Run this command to publish to{" "}
             <StyledLink
-              href="https://vercel.com/docs/cli"
+              href={DeployTargetDocs[deployTarget]}
               target="_blank"
               rel="noreferrer"
             >
-              Vercel CLI
+              {humanizeString(deployTarget)}
             </StyledLink>{" "}
-            to publish your project on Vercel.
           </Text>
         </Grid>
+
+        <Select
+          fullWidth
+          css={{ zIndex: Number.MAX_SAFE_INTEGER.toString() }}
+          id={id}
+          value={deployTarget}
+          options={Object.keys(DeployTargets)}
+          getLabel={(value) => humanizeString(value as string)}
```

Comment:
why `as`

Reply:
Just to humanize the labels, the keys from here
![Screenshot 2023-09-26 at 17 25 39](https://github.com/webstudio-is/webstudio/assets/11075561/7b7de978-ebef-47c2-96cd-831ab85aa089)

But in the UI, we should show a human version of it. LIke, if it is `cloudfare-worker` something like that in future.

Reply:
Try use `as const` as I wrote in other comments, look strange, should already be a string